### ISSUE TITLE
Braille not refreshing properly in editable text fields in Explorer shortcut Properties dialog

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3708]
**Reported by aliminator on 2013-12-10 09:53**
This appears with both tethered to focus and Review.
Pressing alt+enter on the Desktop, e.g. the NVDA shortcut to edit its parameters. Adding e.g. a Parameter does not refresh the Braille Display e.g. 
the new entered value is not shown on the Display.
The Display is updated as soon as Cursor keys are moved.

**Comment by bhavyashah** [comment]
@jcsteh confirms the reported issue in https://github.com/nvaccess/nvda/issues/3708#issuecomment-155313961. Does this still occur? If so, please assign a priority you deem appropriate to this ticket.

**Comment by Adriani90** [comment]
@aliminator, @halim can you still reproduce this issue?

**Comment by Adriani90** [comment]
@ruifontes could you please test if this is reproducible for you?

**Comment by Adriani90** [comment]
@dkager could you please help us in testing if this is still an issue in NVDA 2019.1.1 or Alpha? Unfortunately I cannot test because I do not have a braille display at hand.
cc: @leonardder

**Comment by dkager** [comment]
I can't reproduce this in Windows 10, but that doesn't say anything about Windows 7 (as per the original report).

**Comment by Adriani90** [comment]
@nvdaes, @hozosch can you reproduce this on Windows 7?

**Comment by lukaszgo1** [comment]
I cannot reproduce it with Focus 40 Blue 4th gen on Windows 7 Pro x64 with NVDA version alpha-17311,0d29265e

**Comment by hozosch** [comment]
Well, with my windows 7 professional 64 bit system, I can neither reproduce it with the latest stable version nor the latest snapshot using my handy tech active star AS4-16016

**Comment by nvdaes** [comment]
I can't reproduce this on Windows 7 Professional 64 with NVDA 2019.1.1, testing with my Ecobraille Plus Display tethered to focus, automatic tether and review cursor.

**Comment by Adriani90** [comment]
Thank you all for reporting back. I am closing this as works for me.
